### DISCUSSION TITLE
How to disable "Tab to restart this app and go full screen" button/pop-up

### DISCUSSION THREAD

**Discussion by nattaponx** [koreader/koreader#10877] [Q&A] [answered]
Is there a way to disable "Tab to restart this app and go full screen" button/pop-up at the bottom right? It shows everytime I rotate my android e-reader (ONYX Boox Page).
![IMG_2936](https://github.com/koreader/koreader/assets/10026349/03aad0a4-7d5d-4f69-88f7-2eeb16782434)

**Comment by Frenzie** [comment, marked as answer]
You're probably better off asking that somewhere over on MobileRead. That comes from the OS.
